virtualenv_tools.py

```diff
@@ -178,20 +179,23 @@ def _update_pth_file(pth_filename, orig_path):
         changed = True
         relto_original = os.path.relpath(val, orig_path)
         # lib/pythonX.X/site-packages
-        relto_pth = os.path.join('../../..', relto_original)
+        relto_pth = os.path.join(
+            '..' if is_pypy else '../../..',
+            relto_original
+        )
```

Comment:
Deserves a comment.
Note that you've separated the `../../..` from its comment.

- [ ] fixed